plugins/modules/snap.py

```diff
@@ -52,9 +54,9 @@
         description:
             - Define which release of a snap is installed and tracked for updates.
               This option can only be specified if there is a single snap in the task.
+            - If not passed, the C(snap) command will default to I(stable).
```

Comment:
I'm not sure what I originally wanted to comment - apparently I decided against it but forgot to delete that part of the review - but now I noticed something:
```suggestion
            - If not passed, the C(snap) command will default to C(stable).
```
`C()` should be used for values. Alternatively use the new `V()`:
```suggestion
            - If not passed, the C(snap) command will default to V(stable).
```